packages/modules/src/modules/banners/ausElectionMoment/AusElectionMomentBanner.stories.tsx

```diff
@@ -1,126 +0,0 @@
-import React from 'react';
```

Comment:
It would be good to keep this stories file so we know what this banner looked like

Reply:
Yes, makes sense for reference, will add back now. Just figuring out how to do this!